Frontend/nutri-pal/src/components/dashboard/ProgressBar.tsx

```diff
@@ -22,7 +22,7 @@ export const ProgressBar = ({ max, actual }: ProgressProps) => {
         if (kcal < actual) {
             setTimeout(() => setKcal(n => n + 1), 2);
         }
-        if (width <= 100) {
+        if (width <= 100 && max !== 0) {
             setWidth((kcal / max) * 100);
         } else {
```

Comment:
Quick Fix not on this Pull Request

- On the else condition change it to `else if (width >= 100 && max !== 0)` so that the progress bar is not always red color, but green at first and if it surpasses changes to red